### ISSUE TITLE
Transform Useful Scripts into CNTO Additions or Composition Modules

### ISSUE THREAD

**Issue by JamesTheClarke** [CntoDev/Useful-Scripts#4]
Not sure how feasible this is but it would be cool if R&D could go through the various scripts stored here and identify those that could be turned into either EDEN composition module (firestorm script maybe?) or turned into submods for CNTO Additions.

**Comment by freghar** [comment]
This may be a duplicate of https://github.com/CntoDev/ideas/issues/3 .

**Comment by JamesTheClarke** [comment]
Yeah, definitely. Though I'd leave it open in this repo for PR links, however, I won't add it to the projects page to avoid duplicates there. Cool?
